src/test/java/picard/fingerprint/CheckFingerprintTest.java

```diff
@@ -181,6 +185,62 @@ public void testIdentifyContaminant() {
         Assert.assertTrue(FingerprintChecker.calculateMatchResults(fpContaminant, fpContaminated).getLOD() < -4D);
     }
 
+    @Test
+    public void testFPToVC() throws IOException {
+
+        final File Na12892 = new File(TEST_DATA_DIR, "NA12892.over.fingerprints.shifted.for.crams.r1.sam");
+        final File Na12891 = new File(TEST_DATA_DIR, "NA12891.over.fingerprints.shifted.for.crams.r1.sam");
+
+       final FingerprintChecker checker = new FingerprintChecker(SHIFTED_HAPLOTYPE_DATABASE_FOR_TESTING);
+
+        checker.setLocusMaxReads(100);
+        final Fingerprint fingerprint = checker.identifyContaminant(Na12892.toPath(), 1).get("NA12892");
+        Assert.assertNotNull(fingerprint);
+
+        final File tempFile = File.createTempFile("testWriteFingerprint",".vcf");
+        tempFile.deleteOnExit();
+
+        FingerprintUtils.writeFingerPrint(fingerprint,tempFile,SHIFTED_REFERENCE,
+                "NA12892",null);
+
+        final Fingerprint NA12892FromVCF = checker.fingerprintFiles(Collections.singleton(tempFile.toPath()), 1, 1, TimeUnit.DAYS)
+                .values().stream()
+                .reduce((a, b) -> {
+                    a.merge(b);
```

Comment:
Would it be odd for `Fingerprint.merge()` to return itself? that would make streaming ops like this a bit cleaner to write.

Reply:
good idea.